esp/esp/program/modules/handlers/volunteermanage.py

```diff
@@ -61,6 +63,7 @@ def module_properties(cls):
     @needs_admin
     def volunteering(self, request, tl, one, two, module, extra, prog):
         context = {}
+        response = None
```

Comment:
you shouldn't need this since it doesn't get referred to anywhere outside the if block where it actually gets set.